scripts/verify.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+final_ret=0
+
+lsmod | grep cfg80211
+status=$?
+if [ $status -eq 0 ]; then
+  echo "Module cfg80211 already installed"
+else
+  echo "Installing Module cfg80211"
+  sudo modprobe cfg80211
+fi
+
+lsmod | grep vwifi
```

Comment:
You can even write dow a bash function to detect and manipulate the kernel module.
See https://github.com/sysprog21/kvm-host/blob/master/scripts/common.sh